vita3k/app/src/app_init.cpp

```diff
@@ -319,10 +319,18 @@ bool init(EmuEnvState &state, Config &cfg, const Root &root_paths) {
         window_type |= SDL_WINDOW_ALLOW_HIGHDPI;
         state.dpi_scale = ddpi / 96;
     }
+#elif defined(__APPLE__)
+    window_type |= SDL_WINDOW_ALLOW_HIGHDPI;
+    state.dpi_scale = 2; // It is fixed value on macOS.
```

Comment:
Does using the windows/linux approach (calling SDL_GetDisplayDPI then dividing it by 96) not return 2?

Reply:
macOS's SDL_GetDisplayDPI returns display's real ppi (in my case around 255: 3024x1964 14.2").

Reply:
This part seems to need modification. On Mac, external monitors can be connected in regular mode instead of HiDPI mode, and in that case, dpi_scale should be 1.

On displays connected in HiDPI mode, drawableWidth and drawableHeight will be twice DEFAULT_RES_WIDTH/HEIGHT, but on normal displays, drawableWidth and drawableHeight will be equal to DEFAULT_RES_WIDTH/HEIGHT.

Mac's dpi_scale should be indirectly obtained through the ratio of these values.

Reply:
I made a new PR with fixes because of my mistake.
Now it calculates the scale when the display changes, so it works well in environments with both LoDPI and HiDPI screens.